content/post/2025-05-08-social-media/index.md

```
@@ -0,0 +1,207 @@
+
+## This is really fun.
+
+You should try it. It is quite fun.
+
+## WHAT DOES IT ALL MEAN?
+
+I have been thinking a lot about what all this means. We did something that on the face seems really silly, and it turned out to actually be a performance enhancer. It reminds me that we have no idea what is happening in these lil black box machines. Turns out the context matters.
+
+**My pet theory is that we are speed-running early 2000s enterprise software development lifecycle and work style.**
+
+First it was waterfall (2000, 2001). Now we have added social media (2004, 2008). Next we will probably add work group chat (IRC: 2002-2004, Campfire (2007-2012), and eventually Slack (2013 -...)). I have a stub MCP server you can peep [here](https://github.com/2389-research/workchat-mcp).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)